consultation_analyser/pipeline/processing.py

```diff
@@ -43,14 +45,16 @@ def get_llm_backend(llm_identifier: Optional[str] = None):
         )
 
 
-def process_consultation_themes(consultation, topic_backend=None, llm_backend=None):
+def process_consultation_themes(
+    consultation: Consultation,
+    topic_backend: Optional[TopicBackend] = None,
+    llm_identifier: Optional[str] = None,
```

Comment:
This is breaking encapsulation — could we do it another way? Maybe by initialising sagemaker at the point of usage within the backend class?

Reply:
Perhaps we could subclass `SagemakerEndpoint` and stick the decorator in there?